<launch>
  <node pkg="camera1394" type="camera1394_node" name="camera1394_node" >
    <rosparam file="$(find dcsl_vision_tracker)/config/miabot_camera.yaml" />
  </node>

  <node pkg="image_proc" type="image_proc" name="image_proc" >
    <remap from="image" to="camera/image_raw" />
  </node >
	
  <node pkg="dcsl_vision_tracker" type="background_generator.py" name="background_generator" args="mono8" >
    <param name="image_name" value="$(find dcsl_vision_tracker)/config/miabot_background.png" />
    <remap from="image" to="camera/image_rect" />
  </node>
</launch>